fmralign/alignment_methods.py

```diff
@@ -465,3 +467,239 @@ def fit(self, X, Y):
     def transform(self, X):
         """Transform X using optimal coupling computed during fit."""
         return X.dot(self.R)
+
+
+class IndividualizedNeuralTuning(Alignment):
+    """
+    Method of alignment based on the Individualized Neural Tuning model, by Feilong Ma et al. (2023).
+    It works on 4D fMRI data, and is based on the assumption that the neural response to a stimulus is shared across subjects.
+    It uses searchlight/parcelation alignment to denoise the data, and then computes the stimulus response matrix.
+    See article : https://doi.org/10.1162/imag_a_00032
+    """
+
+    def __init__(
+        self,
+        template="pca",
+        decomp_method=None,
+        n_components=None,
+        alignment_method="searchlight",
+        n_jobs=1,
+    ):
+        """
+        Initialize the IndividualizedNeuralTuning object.
+        Parameters:
+        --------
+
+        - template (str): The type of template to use for alignment. Default is "pca".
+        - decomp_method (str): The decomposition method to use. Default is None.
+        - alignment_method (str): The alignment method to use. Can be either "searchlight" or "parcelation", Default is "searchlight".
+        - n_components (int): The number of latent dimensions to use in the shared stimulus information matrix. Default is None.
+        - n_jobs (int): The number of parallel jobs to run. Default is -1.
+
+        Returns:
+        --------
+        None
+        """
+
+        self.n_s = None
```

Comment:
```suggestion
        self.n_samples = None
```

Reply:
s is for subjects here

```suggestion
        self.n_subjects = None
```

Reply:
Sure, but I'm asking you to give the name, because the code is meant to be read by people whio have a hard time understanding the setting. 
For the same reasons, in sklearn, `n_samples` is called `n_samples`, not `n_s`.